jenkins-x-bdd-vault.yml

```diff
@@ -21,7 +21,9 @@ pipelineConfig:
           - name: ci
             steps:
               - name: setup
-                command: bdd/boot-vault/setup.sh
+                command: bdd/setup.sh
+                args:['bdd/boot-vault', 'bdd-config']
```

Comment:
Needs to be a space after the `:`- same in `jenkins-x-bdd-local.yml`.

Reply:
yap. I think we need to improve the message return when the pipeline runner fails

Reply:
Metapipeline helps with that to some extent.